scripts/stackstorm.js

```diff
@@ -106,7 +106,13 @@ module.exports = function(robot) {
 
     request.get()(
       function(err, resp, body) {
-        var parsed_body, success;
+        var parsed_body, success, error_msg_prefix;
+
+        error_msg_prefix = 'Failed to retrieve commands from ' + env.ST2_API + ': ';
```

Comment:
It's actually used in two if blocks (also the one which checks for `success`).

In any case, yeah, using a module level constant or similar would make it nicer.